dbconn/reload_credentials.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 Block, Inc.
```

Comment:
Why wasn't this file modified in place? The git diff is lost, and sucks because `ReloadDSN` is critical function.

Reply:
Another not to recreate files: git history is lost. So if someone wants to see "who wrote this code?", I'll all point to you.